tp1/freeway/Assets/Scripts/BoardManager.cs

```diff
@@ -11,7 +11,11 @@ public class BoardManager : MonoBehaviour {
 	// 2 = both
 	// 3 = white
 	// 4 = end
-	public GameObject chicken;										//Player
+
+	public static float assetsSize = 0.36f;
```

Comment:
I think this changes were merge in master? I'm not sure why this stays as a new change